source/_patterns/_animations.scss

```diff
@@ -0,0 +1,246 @@
+/*
+ *
+ * Attention-Hogger for Buttons
+ * Used by button.scss
+ *
+ */
+
+$small-size: .8;
+$big-size: 1;
+$dark-attention-color: #aa0000;
+$light-attention-color: #ff0000;
+
+@keyframes pulsatingAnimation {
+  0% {
+    transform: scale($small-size);
+    background-color: $dark-attention-color;
+  }
+  20% {
+    transform: scale($big-size);
+    background-color: $light-attention-color;
+  }
+  40% {
+    transform: scale($small-size);
+    background-color: $dark-attention-color;
+  }
+  60% {
+    transform: scale($big-size);
+    background-color: $light-attention-color;
+  }
+  80% {
+    transform: scale($small-size);
+    background-color: $dark-attention-color;
+  }
+  100% {
+    transform: scale($small-size);
+    background-color: $dark-attention-color;
+  }
+}
+
+
+
+/*
+ *
+ * Animated «LIVE»-Badge-Color in OnAir Navigation
+ * Used by onairnav-link.scss
+ *
+ */
+
+@keyframes pulsatingLiveAnimation {
+  25% {
+    background-color: #e31f2b;
+  }
+  50% {
+    background-color: #ed7004;
+  }
+  75% {
+    background-color: #e31f2b;
+  }
+}
+
+
+
+/*
+ *
+ * Shimmer-Animation
+ * Used by shimmer.scss
+ *
+ */
+
+@keyframes shimmeringAnimation {
+  0% {
+    transform: translateX(-100%);
+  }
+  100% {
+    transform: translateX(200%);
+  }
+}
+
+
+
+/*
+ *
+ * Rotates element all around
+ * Used by _mixins.scss, spinner.scss
+ *
+ */
+
+@keyframes rotate360Animation {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+
+
+/*
+ *
+ * Ripple-Effect
+ * Used by _ripple.scss
+ *
+ */
+
+@keyframes rippleShadowAnimation {
+  0% {
+    box-shadow: 0 0 0 rgba(0, 0, 0, .0);
+  }
+  20% {
+    box-shadow: 0 4px 16px rgba(0, 0, 0, .3);
+  }
+  100% {
+    box-shadow: 0 0 0 rgba(0, 0, 0, .0);
+  }
+}
+
+@keyframes rippleAnimation {
+  to {
+    transform: scale(24);
+    opacity: 0;
+  }
+}
+
+
+
+/*
+ *
+ * Waggles the Element to the right
+ * Used by image-carousel.scss
+ *
+ */
+
+@keyframes waggleRightAnimation {
+  0% {
+    background-position: 10%;
+  }
+  50% {
+    background-position: 80%;
+  }
+  100% {
+    background-position: 10%;
+  }
+}
+
+
+
+/*
+ *
+ * Introduces the World to the dark background of a Modal-Overlay
+ * Used by modal.scss
+ *
+ */
+
+@keyframes modalOverlayAnimation {
```

Comment:
Perfekte Möglichkeit für "Fade to Grey"! 
https://www.youtube.com/watch?v=vMPR6Ujop4k